Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM alpine:3.9
+FROM gcr.io/distroless/static
 
 COPY kube-state-metrics /
 
 ENTRYPOINT ["/kube-state-metrics", "--port=8080", "--telemetry-port=8081"]
 
-RUN adduser -D kube-state-metrics
```

Comment:
why remove this?

Reply:
I had initially thought that it wasn't necessary to explicitly create a non-root user in distroless, but that does not seem to be the case.

After some research, I see that I would need to use multi-stage builds. For the sake of making a clean change, this means changing up the Makefiles also a bit and other scripts that depend on it. I'll need to change this PR up significantly.

Reply:
what is the use case for multi-stage builds?

Reply:
Great question! This is the rationale essentially. https://medium.com/@lizrice/non-privileged-containers-based-on-the-scratch-image-a80105d6d341